salt/modules/win_pkg.py

```diff
@@ -1447,6 +1447,11 @@ def install(name=None, refresh=False, pkgs=None, **kwargs):
                 'extra_install_flags': kwargs.get('extra_install_flags')
             }
         }
+    elif len(pkg_params) == 1:
+        # A dict of packages was passed, but it contains only 1 key, so we need
+        # to add the 'extra_install_flags'
+        for pkg in pkg_params:
```

Comment:
remove for loop

Reply:
would it be better as `pkg = next(iter(pkg_params))`? Not sure if there's another, better approach.

Reply:
I like that